catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: "Component"
+metadata:
+  name: 'edx-notes-api'
+  description: "A backend store for edX Student Notes"
+  annotations:
```

Comment:
We should remove `annotations` key altogether
like
https://github.com/openedx/xblock-lti-consumer/blob/8197943d4e4abc6d058c6038500582df47ed493e/catalog-info.yaml